src/test/http_rest_api_handler_test.cpp

```diff
@@ -35,11 +35,15 @@ static const char* config_1 = R"(
     ]
 })";
 
-class ModelControlApi : public TestWithTempDir {};
+class ModelControlApi : public TestWithTempDir {
+public:
+    void SetUp() {
```

Comment:
Check eg here:
https://github.com/openvinotoolkit/model_server/blob/develop/src/test/ensemble_config_change_stress.cpp#L592

You create in L46 config in default path and then if you run the tests simultaneously with jobs > 2 you run into a risk of data race (concurrent remove/modification of file in several tests).

You should create config file in separate directory and then cleanup - you can reuse TestWithTempDir. Example is in ensemble_config_stress_tests.